packages/drivers/odsp-driver/src/test/epochTestsWithRedemption.spec.ts

```diff
@@ -34,6 +34,8 @@ class DeferralWithCallback extends Deferred<void> {
     }
 }
 
+const failOnUnhandledRejection = () => assert.fail("Rejections should not cause unhandled rejection event");
```

Comment:
I think Mocha should fail the tests if there's an unhandled rejection even without this.